src/api.ts

```diff
@@ -513,6 +513,8 @@ export function fernsRouter<T>(
       }
 
       let builtQuery = model.find(query).limit(limit + 1);
+      let total = 0;
+      total = await model.countDocuments(query);
```

Comment:
@joshgachnang We're not currently protecting against some of the queries that might break this specific function (eg. $where). I initially had a check here that would have thrown an error but testing for that specific use case proved to be tricky. I figured we could get this out and then come back to it if it starts to pose an issue?